nf_core/pipeline-template/.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -45,6 +45,6 @@ body:
         * Nextflow version _(eg. 22.10.1)_
         * Hardware _(eg. HPC, Desktop, Cloud)_
         * Executor _(eg. slurm, local, awsbatch)_
-        * Container engine: _(e.g. Docker, Singularity, Conda, Podman, Shifter or Charliecloud)_
+        * Container engine: _(e.g. Docker, Singularity, Conda, Podman, Shifter, Charliecloud, or Apptainer)_
```

```suggestion
        * Container engine: _(e.g. Docker, Singularity/Apptainer, Conda, Podman, Shifter or Charliecloud)_
```
Apptainer is just renamed singularity, no? I would therefor treat it in that way.

---

No not necessarily. They plan to make differences in the future IIRC

---

OK I 'm correct there, but I would rather just have it as a separate thing to follow the same scheme as they are separate binaries (and not confuse others)